tests/transports/epics/test_emission.py

```
@@ -0,0 +1,206 @@
+from fastcs.transports.epics.pva.gui import PvaEpicsGUI
+
+
+class _Alpha(Controller):
+    foo = AttrR(Int())
+
+
+class _Beta(Controller):
+    bar = AttrR(Int())
+
+
+def _api_with_id(controller_class: type[Controller], id: str):
```

Ruff (0.15.13)

[error] Function argument `id` is shadowing a Python builtin

(A002)